rust/connlib/shared/src/messages/client.rs

```diff
@@ -76,6 +76,10 @@ impl ResourceDescriptionCidr {
 /// Description of an internet resource.
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub struct ResourceDescriptionInternet {
+    /// Name of the resource.
+    ///
+    /// Used only for display.
+    pub name: Option<String>,
```

Comment:
Why are we making it optional?

Reply:
Right now, the portal doesn't send it, can change if we decide it always send it.

Reply:
You can set a default value with `serde`. That might make things a bit easier because you don't need to deal with the `Option`?

Reply:
Yeah second the recommendation to default with `serde` -- that way we can override it from the portal in the future.